spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -303,10 +303,8 @@ public SparkContext createSparkContext() {
     for (Object k : intpProperty.keySet()) {
       String key = (String) k;
       String val = toString(intpProperty.get(key));
-      if (!key.startsWith("spark.") || !val.trim().isEmpty()) {
-        logger.debug(String.format("SparkConf: key = [%s], value = [%s]", key, val));
-        conf.set(key, val);
-      }
```

Comment:
The reason why I coded the above like that was to avoid the case that we set the key only, For instance, if you remove this code and you leave spark.yarn.jar empty in a interpreter tab, you cannot run any spark code in a yarn environment. Do you have any other code to avoid that case?